### DISCUSSION TITLE
[sdcard] Can't access usable part of SD Card

### DISCUSSION THREAD

**Discussion by orez-ero** [gsantner/markor#1734] [Q&A] [answered]
I am having a little bit of trouble with Markor now that I have moved from a phone running Android 11 to a Galaxy 53 running Android 12. I can no longer access the public areas of my SD card even though I am pretty sure that proper permissions have been granted. You can see what I do get in the attached screen shots...

![Screenshot_20220531-080744_Markor](https://user-images.githubusercontent.com/106614949/171251684-735198c6-cffe-4170-a54e-7ee2a2c7dd30.jpg)

![Screenshot_20220531-080843_Markor](https://user-images.githubusercontent.com/106614949/171251850-4596769d-618b-41ea-940b-f43cf7c52a98.jpg)

Is this a problem with my setup? Am I doing something wrong? Or might there be an issue with Markor itself?

Thanks in advance for any help you can give me...

**Comment by gsantner** [comment, marked as answer]
I made some improvements for SD-Card support recently. Please see <https://github.com/gsantner/markor/issues/603>.
